src/components/JournalPage/index.jsx

```diff
@@ -45,7 +46,8 @@ class JournalPage extends React.Component {
         <UnauthorizedPage />
       ) : (
         <div className="page">
-          <h3>{this.props.title}</h3>
+          <h1>{this.props.title}</h1>
+          <h5>{this.props.subTitle}</h5>
```

Comment:
It's an accessibility issue to skip header levels as a screen reader will tell users that there are no `<h2>`s, so it assumes there's nothing below that. Headers should be used to designate structure, styling comes second. In this case we'd want the `h5` to be an `h2` and we can style `h2`s (or add a subtitle class) to appropriately fit the space.

Reply:
Good point - i think we should add a subtitle class and make the subtitle h2.